en/components/drag_drop.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Drag Drop Directive
+_description: Use Ignite UI for Angular Drag Drop directive to move items from one place to another.
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular Drag Drop directive, Angular Drag Drop controls
+---
+
+## Drag and Drop
+<p class="highlight">The Ignite UI for Angular Drag and Drop directive enables dragging of elements around the page.</p>
+<div class="divider"></div>
+
+#### Demo
+
+<div class="sample-container loading" style="height:650px">
+    <iframe id="drag-drop-sample-iframe" src='{environment:demosBaseUrl}/drag-drop-sample' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<br/>
+<div>
+<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="drag-drop-sample-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+<div class="divider--half"></div>
+
+### Drag Directive
+
+When an element inside your Angular application needs to be dragged from one place to another the [`igxDrag`]({environment:angularApiUrl}/classes/igxdragdirective.html) directive can be used to achieve this behavior. In combination with the [`igxDrop`]({environment:angularApiUrl}/classes/igxdropdirective.html) directive the placing of the dragged element can be done as well to have fully interactive application.
+
+#### Basic configuration
+
+The [`igxDrag`]({environment:angularApiUrl}/classes/igxdragdirective.html) directive can be applied on any DOM element by just adding it to its template.
+
+```html
+<div igxDrag>Drag me</div>
+```
+
+#### DOM Behavior
+
+By default a drag operation starts when the end user swipes at least 5 px in any direction. Otherwise the interaction is considered as a click and the [`dragClicked`]({environment:angularApiUrl}/classes/igxdragdirective.html#dragclicked) event is emitted.
+
+When dragging occurs a drag ghost element is spawned and moves along with the mouse cursor or touch interaction. The original element is still present, but it can be hidden automatically when dragging starts with the [`hideBaseOnDrag`]({environment:angularApiUrl}/classes/igxdragdirective.html#hidebaseondrag) input.
+
+The dragging can be canceled by setting the `cancel` property of the [`dragStart`]({environment:angularApiUrl}/classes/igxdragdirective.html#dragstart) event to `true`. This will cancel the default dragging logic.
```

Comment:
Provide reference to the `cancel`  property of the IDragStartEventArgs interface:
[`cancel`] ({environment:angularApiUrl}/interfaces/idragstarteventargs.html#cancel)